### PR TITLE
Start 1.2.0 development cycle

### PR DESCRIPTION
**Pull request by ericgribkoff**
v1.1.0 is tagged and pushed to Maven.

### CODE REVIEW

**Comment by jyane** [comment]
1.1.0 is already available on maven central, I've checked sync was done.
Updating examples will be follow up?

**Review by jyane — DISMISSED** [review]

**Comment by ericgribkoff** [comment]
Examples bumped to 1.1.0 now too

File: examples/WORKSPACE [hunk 1]
```diff
@@ -12,6 +12,6 @@ grpc_java_repositories()
 
 maven_jar(
     name = "io_grpc_grpc_java_api_checker",
-    artifact = "io.grpc:grpc-java-api-checker:1.0.0",
+    artifact = "io.grpc:grpc-java-api-checker:1.1.0",
     sha1 = "68fe2fc82e1f673b042fb7192a3724d1464d0582",
```
Comment (jyane) [inline examples/WORKSPACE]: is it `d0c4b2979fbd7127d0c3e587f9076ead9457d344` ?

Comment (ericgribkoff) [reply examples/WORKSPACE]: fixed

**Review by jyane — APPROVED** [review]
Thank you! LGTM!

**Pull request merged**